product_docs/docs/edbcloud/beta/administering_cluster/02_postgres_access.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: "Managing Postgres Access"
+---
+
+The `edb_admin` database role and `edb_admin` database created during the _Create Cluster_ process should not be used by your application. Instead, create a new database role and a new database, which provides a high level of isolation in Postgres. If multiple applications are using the same cluster, each database can also contain multiple schemas, essentially a namespace in the database. If strict isolation is needed, use a dedicated cluster or dedicated database. If that strict isolation level is not required, a single database can be deployed with multiple schemas. Please refer to the [PostgreSQL documentation](https://www.postgresql.org/docs/current/ddl-priv.html) on Privileges to further customize ownership and roles to your requirements. 
+
+To create a new role and database, first connect using `psql`: 
+
+```
+psql -W "postgres://edb_admin@xxxxxxxxx.xxxxx.edbcloud.io:5432/edb_admin?sslmode=require"
+```
+
+## One Database with One Application 
+
+For one database hosting a single application: 
+
+1. Create a new database role. For example,
```

Comment:
@gbartolini: You had some comments about a necessary `GRANT` statement here - can you drop those comments in to this PR?

Reply:
@ba-edb @gbartolini 

additional GRANT statements for edb_admin added as part of PR 1752: https://github.com/EnterpriseDB/docs/pull/1752